dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -251,9 +252,9 @@ private long countAllPublished() {
     public List<String> getWorkflowsDescriptorTypes(long entryId) {
         List result = this.currentSession().getNamedQuery("Entry.findWorkflowsDescriptorTypes").setParameter("entryId", entryId).getResultList();
         if (!result.isEmpty()) {
```

Comment:
With the `map`, you don't need the `if...else` any more. If result is empty, calling stream.map on it wil return an empty map.